tests/suites/test_suite_bignum_random.function

```diff
@@ -155,6 +156,61 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mpi_mod_random_values( int min, char *max_hex )
+{
+    /* Same RNG as in mpi_core_random_basic */
+    mbedtls_test_rnd_pseudo_info rnd_core = {
+        {'T', 'h', 'i', 's', ' ', 'i', ',', 'a',
+         's', 'e', 'e', 'd', '!', 0},
+        0, 0};
+    mbedtls_test_rnd_pseudo_info rnd_mod_raw;
+    memcpy( &rnd_mod_raw, &rnd_core, sizeof( rnd_core ) );
+    mbedtls_mpi_uint *R_core = NULL;
+    mbedtls_mpi_uint *R_mod_raw = NULL;
+    mbedtls_mpi_mod_modulus N;
+    mbedtls_mpi_mod_modulus_init( &N );
+
+    TEST_EQUAL( mbedtls_test_read_mpi_modulus( &N, max_hex,
+                                               MBEDTLS_MPI_MOD_REP_MONTGOMERY ),
+                0 );
+    ASSERT_ALLOC( R_core, N.limbs );
+    ASSERT_ALLOC( R_mod_raw, N.limbs );
```

Comment:
Should we test the input's at this stage? 

-  `min < N`
-  `R_Core` is not aliased to `N.p`
- `mbedtls_test_rnd_pseudo_rand` is not NULL?

Reply:
`min` and `N` are inputs, so they don't need to be validated.

None of `mbedtls_mpi_core_random`, `mbedtls_mpi_mod_raw_random` or `mbedtls_mpi_mod_random` allow the output to be aliased with the upper bound (which is the modulus for the mod-raw and mod functions).

`mbedtls_test_rnd_pseudo_rand` is from the test code, it doesn't need to be validated.